### PR TITLE
Use typed service POJOs in tests and custom handler

### PR DESCRIPTION
**Pull request by mofterdinger**

### CODE REVIEW

File: srv/src/main/java/my/bookshop/handlers/CatalogServiceHandler.java [hunk 1]
```diff
@@ -58,15 +59,15 @@
...
+	private final ReviewService.Draft reviewService;
 
 	private final Messages messages;
 	private final FeatureTogglesInfo featureToggles;
 	private final RatingCalculator ratingCalculator;
 	private final CqnAnalyzer analyzer;
 
-	CatalogServiceHandler(PersistenceService db, @Qualifier(ReviewService_.CDS_NAME) DraftService reviewService,
-			Messages messages, FeatureTogglesInfo featureToggles, RatingCalculator ratingCalculator, CdsModel model) {
+	CatalogServiceHandler(PersistenceService db, ReviewService.Draft reviewService, Messages messages,
```
Comment (beckermarc) [inline srv/src/main/java/my/bookshop/handlers/CatalogServiceHandler.java:69]: I think this class doesn't actually require the `DraftService` APIs.

```suggestion
	CatalogServiceHandler(PersistenceService db, ReviewService reviewService, Messages messages,
```

File: srv/src/main/java/my/bookshop/handlers/AdminServiceAddressHandler.java [hunk 2]
```diff
@@ -53,9 +53,9 @@ public class AdminServiceAddressHandler implements EventHandler {
 
 	// We are mashing up the AdminService with two other services...
 	private final PersistenceService db;
-	private final CqnService bupa;
+	private final ApiBusinessPartner bupa;
 
-	AdminServiceAddressHandler(PersistenceService db, @Qualifier(ApiBusinessPartner_.CDS_NAME) CqnService bupa) {
+	AdminServiceAddressHandler(PersistenceService db, ApiBusinessPartner bupa) {
```
Comment (beckermarc) [inline srv/src/main/java/my/bookshop/handlers/AdminServiceAddressHandler.java]: Readd `@Qualifier` to fix tests

**Review by beckermarc — COMMENTED** [review]
I think in `AdminServiceAddressITestBase` there is also a service that could be replaced.

File: srv/src/main/java/my/bookshop/handlers/CatalogServiceHandler.java (hunk 1 above)
Comment (mofterdinger) [reply srv/src/main/java/my/bookshop/handlers/CatalogServiceHandler.java]: changed

**Comment by mofterdinger** [comment]
> `AdminServiceAddressITestBase`

thanks, done.

**Review by beckermarc — APPROVED** [review]

**Pull request merged**